datasets/benkagg/handelsregisterkvk/v2.2.0.json

```diff
@@ -0,0 +1,826 @@
+            "description": "Vrije tekst om een Locatie nader te kunnen duiden. Bijvoorbeeld bij bedrijfsverzamelgebouwen met meerdere vestigingen"
+          },
+          "volledigAdres": {
+            "type": "string",
+            "title": "Volledig adres",
+            "description": "Samengesteld adres dat wordt afgeleid van de overige adresgegevens"
+          },
+          "straatnaam": {
+            "type": "string",
+            "title": "Straat naam",
+            "description": "Officiële naam openbare ruimte (80 tekens)"
+          },
+          "huisnummer": {
+            "type": "integer",
+            "title": "Huisnummer",
+            "description": "Een door het bevoegde gemeentelijke orgaan ten aanzien van een adresseerbaar object toegekende nummering"
+          },
+          "huisletter": {
+            "type": "string",
+            "title": "Huisletter",
+            "description": "Een door het bevoegde gemeentelijke orgaan ten aanzien van een adresseerbaar object toegekende toevoeging aan een huisnummer in de vorm van een alfanumeriek teken"
+          },
+          "huisnummerToevoeging": {
+            "type": "string",
+            "title": "Huisnummertoevoeging",
+            "description": "Een door het bevoegde gemeentelijke orgaan ten aanzien van een adresseerbaar object toegekende nadere toevoeging aan een huisnummer of een combinatie van huisnummer en huisletter"
+          },
+          "postcode": {
+            "type": "string",
+            "title": "Postcode",
+            "description": "De door PostNL vastgestelde code bestaande uit 4 cijfers en 2 letters (1234AB)"
+          },
+          "plaats": {
+            "type": "string",
+            "title": "Plaats naam",
+            "description": "Officiële naam woonplaats"
+          },
+          "straatHuisnummerBuitenland": {
+            "type": "string",
+            "title": "Buitenland straat huisnummer",
+            "description": "Het straat/huisnummer is een combinatie van de straat en het huisnummer van het buitenlandse adres"
+          },
+          "postcodePlaatsBuitenland": {
+            "type": "string",
+            "title": "Buitenland plaats postcode",
+            "description": "De postcode/woonplaats is de combinatie van een eventuele postcode en woonplaats van het buitenlandse adres"
+          },
+          "regioBuitenland": {
+            "type": "string",
+            "title": "Buitenland regio naam",
+            "description": "Regio is een deel van het land (streek, provincie, etc.) van het buitenlandse adres"
+          },
+          "landBuitenland": {
+            "type": "string",
+            "title": "Buitenland naam",
+            "description": "De naam van het land waar het adres zich bevindt"
+          }
+        },
+        "title": "Bezoeklocatie locatiegegevens",
+        "description": "De locatiegegevens van de bezoekLocatie"
+      },
+      "bezoekHeeftBagNummeraanduidingVes": {
```

Comment:
Obv van de naam is dit een boolean. Dat strookt niet met de beschrijving.
 `"nummeraanduidingBezoeklocatie"`

Reply:
Een auto heeft wielen en een auto heeft de eigenschap dat ie wielen heeft. Maar een auto heeft geen heeftwielen.